code/modules/admin/player_panel/actions/physical.dm

```diff
@@ -183,8 +183,12 @@
 	for (var/obj/item/I in target)
 		if(params["drop_items"])
 			target.drop_inv_item_to_loc(I, target.loc, FALSE, TRUE)
-		else
-			qdel(I)
+			continue
+
+		if(istype(I, /obj/item/card/id))
+			continue
```

Comment:
With this here, after the drop_items check, if you're in drop_items mode, the ID card will be dropped to the ground. Should ID cards only prevent deletion, but otherwise get dropped if in that mode??

Reply:
I suppose in most cases it's better if they keep their ID

Reply:
I'll fix it right now